packages/components/nodes/GoogleDocs/GoogleDocs.ts

```diff
@@ -0,0 +1,295 @@
+    label: string;
+    name: string;
+    type: NodeType;
+    description?: string;
+    version: number;
+    icon: string;
+    incoming: number;
+    outgoing: number;
+    actions?: INodeParams[];
+    credentials?: INodeParams[];
+    inputParameters?: INodeParams[];
+
+    constructor() {
+        this.label = 'GoogleDocs';
+        this.name = 'googleDocs';
+        this.icon = 'gdocs.svg';
+        this.type = 'action';
+        this.version = 1.0;
+        this.description = 'Execute GoogleDocs operations';
+        this.incoming = 1;
+        this.outgoing = 1;
+
+        this.actions = [
+            {
+                label: 'Operation',
+                name: 'operation',
+                type: 'options',
+                options: [
+                    {
+                        label: 'Create New Document',
+                        name: 'create',
+                        description: 'Create a new document'
+                    },
+                    {
+                        label: 'Get All Values',
+                        name: 'getAll',
+                        description: 'Get all values from a document'
+                    },
+                    {
+                        label: 'Update a Document',
+                        name: 'update',
+                        description: 'Update a document'
+                    }
+                ],
+                default: 'create'
+            }
+        ] as INodeParams[];
+
+        this.credentials = [
+            {
+                label: 'Credential Method',
+                name: 'credentialMethod',
+                type: 'options',
+                options: [
+                    {
+                        label: 'Google Docs OAuth2',
+                        name: 'googleDocsOAuth2Api'
+                    }
+                ],
+                default: 'googleDocsOAuth2Api'
+            }
+        ] as INodeParams[];
+
+        this.inputParameters = [
+            {
+                label: 'Document Name',
+                name: 'documentName',
+                type: 'string',
+                optional: true,
+                description: 'Name of the document to create. Default to Untitled document.',
+                show: {
+                    'actions.operation': ['create']
+                }
+            },
+            {
+                label: 'Document',
+                name: 'documentId',
+                type: 'asyncOptions',
+                loadMethod: 'getAllDocsFromDrive',
+                hide: {
+                    'actions.operation': ['create']
+                }
+            },
+
+            /**
+             *  batch update
+             */
+
+            {
+                label: 'Requests',
+                name: 'requests',
+                description:
+                    "update a document. You can simply add one reequest data or add multiple request data. If request format is invalid, document won't be updated.",
```

Comment:
Would be better to add a link (https://developers.google.com/docs/api/reference/rest/v1/documents/batchUpdate) so users can take reference on how to write the requests body

Reply:
I added a link. 
This image is how it's displayed
<img width="276" alt="Screen Shot 2022-10-15 at 11 07 40 AM" src="https://user-images.githubusercontent.com/69574727/196001745-2e86ac93-f8a4-448f-b409-e4f20a0beff1.png">